### ISSUE TITLE
Banner is being weird.

### ISSUE THREAD

**Issue by labr0110** [stephyspiccoli/restaurant-website#1]
Your banner goes off to the side when It scales down.

![screen shot 2016-04-25 at 12 57 35 pm](https://cloud.githubusercontent.com/assets/16653418/14791798/6905cad0-0ae5-11e6-9857-1fcf45b0e8fd.png)

**Comment by scallsen** [comment]
Yeah, I'm finding the cta button position is not working as intended as well. (Firefox)
<img width="1093" alt="screen shot 2016-04-25 at 12 58 28 pm" src="https://cloud.githubusercontent.com/assets/16653087/14791837/939ec814-0ae5-11e6-9951-9681363c5dd5.png">
